.changes/next-release/feature-AWSCommonRuntimeHTTPClient-dda0ac9.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "__AWS Common Runtime HTTP Client__", 
+    "contributor": "", 
+    "type": "feature", 
+    "description": "Adds support for Https system properties"
```

Comment:
Can we be more specific about what system properties we have added as part of this change? Same as other changelog entries

Reply:
Updated to add descriptions